source/clear-linux/tutorials/multi-boot/multi-boot-ubuntu.rst

```diff
@@ -109,15 +106,15 @@ Install Ubuntu\* 16.04 LTS Desktop
 
       .. note:: The root partition UUID used below is unique to this example.
 
-      .. code-block:: console
+      .. code-block:: bash
 
-         # cd /boot/efi/loader/entries
+         sudo cd /boot/efi/loader/entries
 
-         # vi ubuntu.conf
+         sudo vi ubuntu.conf
```

Comment:
Consolidate lines 111 and 113 to "sudoedit /boot/efi/loader/entries/ubuntu.conf"